projects/gui/src/gamedatabasedlg.cpp

```diff
@@ -436,6 +452,7 @@ void GameDatabaseDialog::gameSelectionChanged(const QModelIndex& current,
 	ui->m_resultLabel->setText(game.tagValue("Result"));
 
 	m_gameViewer->setGame(&game);
+	m_game = game;
```

Comment:
Now that you've created the `m_game` object you don't need `game` anymore and will save a copy operation.

Reply:
Ah, thank you. Reusing `m_game` works, because `PgnGame::read()` calls `PgnGame::clear()`. I did not look into the details of PgnGame then. So I used a fresh copy.